src/renderer/features/assets/AssetsPortfolio/ui/AssetsPortfolioView.tsx

```diff
@@ -0,0 +1,67 @@
+import { useUnit } from 'effector-react';
+
+import { useI18n } from '@app/providers';
+import { Icon, BodyText, FootnoteText } from '@shared/ui';
+import { priceProviderModel } from '@entities/price';
+import { walletModel } from '@entities/wallet';
+import { Wallet, WalletType } from '@shared/core';
+import { AssetsListView } from '@entities/asset';
+import { TokenBalanceList } from './TokenBalanceList';
+import { TokenBalance } from './TokenBalance';
+import { portfolioModel } from '../model/portfolio-model';
+
+const getColStyle = (wallet?: Wallet) => {
+  switch (wallet?.type) {
+    case WalletType.WATCH_ONLY:
+      return 'grid-cols-[1fr,100px,105px]';
+    case WalletType.PROXIED:
+      return 'grid-cols-[1fr,100px,105px,30px]';
+    default:
+      return 'grid-cols-[1fr,100px,105px,50px]';
+  }
+};
```

Comment:
What about this one?
```suggestion
const getColStyle = (wallet?: Wallet): string => {
  const colStyleMap = {
    [WalletType.WATCH_ONLY]: 'grid-cols-[1fr,100px,105px]',
    [WalletType.PROXIED]: 'grid-cols-[1fr,100px,105px,30px]',
    default: 'grid-cols-[1fr,100px,105px,50px]',
  };

  return colStyleMap[wallet?.type] || colStyleMap.default;
};
```